packages/mongo/mongo_driver.js

```diff
@@ -1032,9 +1034,10 @@ Cursor.prototype.observeChanges = function (callbacks, options = {}) {
 };
 
 MongoConnection.prototype._createSynchronousCursor = function(
-    cursorDescription, options) {
+    cursorDescription, options = {}) {
   var self = this;
-  options = _.pick(options || {}, 'selfForIteration', 'useTransform');
+  const { selfForIteration, useTransform } = options; 
```

Comment:
You could unpack this directly in the arguments list.

Reply:
But `options` is reused later on
https://github.com/harryadel/meteor/blob/fd745ffb3f759144292038434a093acbfa162f52/packages/mongo/mongo_driver.js#L1087